cobc/tree.c

```diff
@@ -2782,18 +2782,175 @@ cb_build_alphanumeric_literal (const void *data, const size_t size)
 }
 
 cb_tree
-cb_build_national_literal (const void *data, const size_t size)
+cb_build_gcos_literal(const void *data, const size_t size)
 {
 	cb_tree			l;
 
-	l = CB_TREE (build_literal (CB_CATEGORY_NATIONAL, data, size));
+	l = CB_TREE (build_literal (CB_CATEGORY_ALPHANUMERIC, data, size));
 
 	l->source_file = cb_source_file;
 	l->source_line = cb_source_line;
 
 	return l;
 }
 
+cb_tree
```

Comment:
For those new/adjusted functions below: possibly call the `cb_build_literal_by_category` as well instead of creating the tree the same way?